src/cli/Import.cpp

```diff
@@ -83,7 +83,7 @@ int Import::execute(const QStringList& arguments)
 
     QString errorMessage;
     Database db;
-    db.setKdf(KeePass2::uuidToKdf(KeePass2::KDF_ARGON2));
+    db.setKdf(KeePass2::uuidToKdf(KeePass2::KDF_ARGON2D));
```

Comment:
The patch is overall just a cherry-pick of 9a96124040e2be97c51d3a06df8966ab09e6c668, but in addition, I had to change this line, which doesn't exist anymore on develop.